frontend/src/assets/stylesheets/components/footer.sass

```diff
@@ -0,0 +1,73 @@
+.footer
+  padding: 30px 0px 20px
+  margin-right: 40px
+
+  a
+    color: #A0A0A0
+    text-decoration: none
+    &:hover
+      color: #48bd66
+
+  .social-links
+    display: flex
+    justify-content: space-evenly
+    padding: 0
+    list-style-type: none
+
+    max-width: 1250px
+    margin: 16px auto
+
+    li
+      a
+        display: block
+        svg
+          margin: 14px
+          height: 22px
+
+          path
+            fill: #A0A0A0
+            transition: fill $transition-duration $transition-function
+        &:hover
+          &.-linkedin
+            svg
+              path
+                fill: #0077b5
+
+          &.-medium
+            svg
+              path
+                fill: #1c9963
+
+          &.-youtube
+            svg
+              path
+                fill: red
+
+          &.-behance
+            svg
+              path
+                fill: #1769ff
+
+          &.-dribbble
+            svg
+              path
+                fill: #ea4c89
+
+          &.-twitter
+            svg
+              path
+                fill: #1da1f2
+
+          &.-instagram
+            svg
+              path
+                fill: #e1306c
+
+  &__down
```

Comment:
What `down` means here?

Reply:
This is just the down part of the footer, can't imagine better name for it

Reply:
Name hasn't changed :)

Reply:
Yea you are right, i must forgot about change the name